cip-drep-metadata/README.md

```diff
@@ -0,0 +1,231 @@
+
+`picture`
+- Optional 
+- A base 64 encoded profile picture
+- Moderation of this image must be handled on the client side to comply with their TOS
+- This SHOULD be treated as the profile picture of the individual
+
+`objectives`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of what the person believes nd what they want to achieve as a DRep
+
+`motivation`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of why they want to be a DRep, what personal and professional experiences have they had that have driven them to 
+
+`qualifications`
+- Optional
+- A freeform text field with a maximum of 1000 characters 
+- A space to list the qualifications that the subject of this metadata has that are relevant to being a DRep
+
+`identity`
+- Compulsory
+- A link to a social media profile where the person must reference their DRep ID in their profile.
+- This will be used by people reviewing this DRep to prove and verify that the person described in the metadata is the same as the person who set up the social media profile.
+- Tooling providers SHOULD warn people that none of the information is verified by the tool and they should DYOR
+- Tooling providers making metadata SHOULD provide some information about why this is important.
+
+PKI cryptographic solutions were also considered in the place of the current solution however they were generally considered too complex to implement for minimum viable tooling. There were also other issues with cryptographic forms of identification for MVPs:
+1. solutions that involve a public/private key setup still require people verifying the identity of a DRep to know what the authentic pulic key is.
+2. specifying the use of a verification service such as [Keybase](https://keybase.io/) would lead to centralisation and therefore reliance on a 3rd party for DReps.
+
+`links`
+- Optional
+- A list of urls to the social media/ websites associated with the DRep
+- It is up to the client to check where these links go
+- Warning about linking to external links
+
+`donotlist`
+- Optional
+- If not included then the value is assumed to be false
+- A boolean expression
+- A true value means that the DRep does not want to show up in tooling that displays DReps. 
+-- I.e. they do not want to appear in GovTool’s DRep Explorer feature
+
+
+### Application
+DRep metadata must include all compulsory fields to be considered CIP-???? compliant. As this is an extension to CIP-100, all CIP-100 fields can be included within CIP-???? compliant metadata.
```

Comment:
> all CIP-100 fields can be included within CIP-???? compliant metadata.

This confirmed?

Reply:
You are right to question this. The authors field has been explicitly rejected by this CIP.

Reply:
> The authors field has been explicitly rejected by this CIP.

Or you could include it (makes it easier for tooling to comply)
but suggest the field is left blank?

Reply:
ok fixed. Thanks for the advice. https://github.com/Thomas-Upfield/CIPs/blob/drep-metadata/cip-drep-metadata/README.md#witnesses